lib/vendor.rb

```diff
@@ -1,3 +1,35 @@
+require_relative './item'
+
 class Vendor
-  
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
+    @inventory[item] || 0
+  end
+
+
+  def stock(item, amount)
+    @inventory[item] = check_stock(item) + amount
+  end
+
+  def revenue_item1
+    item1 = Item.new({name: 'Peach', price: "$0.75"})
```

Comment:
item1 is hardcoded here, so this helper method isn't dynamic / only works for this type of item.